.github/workflows/ci_workflows.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Check base branch
-      uses: actions/github-script@v6
+      uses: actions/github-script@v7
```

Comment:
I don't remember if github-script from v6 to v7 requires also other updates to go with it. You might have to check the change log.